nautobot/docs/requirements.txt

```diff
@@ -1,5 +1,5 @@
 griffe==0.31.0
-Jinja2==3.1.2
+Jinja2==3.1.3
```

Comment:
does the example_plugin also need this update or is that 2.x-only?

Reply:
I missed it. It's updated now